packages/sign-client/test/shared/connect.ts

```diff
@@ -25,12 +34,15 @@ export async function testConnectMethod(clients: Clients, params?: TestConnectPa
 
   const connectParams: EngineTypes.ConnectParams = {
     requiredNamespaces: params?.requiredNamespaces || TEST_REQUIRED_NAMESPACES,
+    optionalNamespaces: params?.requiredNamespaces || TEST_OPTIONAL_NAMESPACES,
```

Comment:
Sorry just seeing this now, copy-paste error:

```suggestion
    optionalNamespaces: params?.optional || TEST_OPTIONAL_NAMESPACES,
```